baseband/mark5b/base.py

```diff
@@ -348,17 +336,16 @@ class Mark5BStreamWriter(VLBIStreamWriterBase, Mark5BFileWriter):
 
     _frame_class = Mark5BFrame
 
-    def __init__(self, raw, frames_per_second=None, sample_rate=None,
-                 nchan=1, bps=2, header=None, squeeze=True, **kwargs):
+    def __init__(self, raw, sample_rate, nchan=1, bps=2, header=None,
```

Comment:
This makes me wonder whether `sample_rate` should be first generically...

Reply:
From above, `sample_rate` is one of two optional variables in `VLBIStreamReaderBase.__init__`, which is why it's second-to-last.  Of course, we could make `sample_rate` a positional argument, but I like the fact that all the positional arguments are values `VLBIStreamReaderBase` doesn't know what to do with if they weren't explicitly passed.  If you feel having `sample_rate` be the first keyword makes the code more readable, we can change it (though I'd then argue `samples_per_frame`, whenever passed, should go right after).